src/stream/src/executor/hash_agg.rs

```diff
@@ -312,14 +332,33 @@ impl<K: HashKey, S: StateStore> HashAggExecutor<K, S> {
     #[try_stream(ok = StreamChunk, error = StreamExecutorError)]
     async fn flush_data<'a>(
         &mut HashAggExecutorExtra::<S> {
+            ref ctx,
             ref key_indices,
             ref schema,
             ref mut state_tables,
+            ref lookup_miss_count,
+            ref total_lookup_count,
+            ref metrics,
             ..
         }: &'a mut HashAggExecutorExtra<S>,
         state_map: &'a mut EvictableHashMap<K, Option<Box<AggState<S>>>>,
         epoch: u64,
     ) {
+        let actor_id_str = ctx.id.to_string();
+        metrics
+            .agg_lookup_miss_count
+            .with_label_values(&[&actor_id_str])
+            .inc_by(lookup_miss_count.load(Ordering::Relaxed));
+        metrics
+            .agg_total_lookup_count
+            .with_label_values(&[&actor_id_str])
+            .inc_by(total_lookup_count.load(Ordering::Relaxed));
+        lookup_miss_count.store(0, Ordering::Relaxed);
```

Comment:
[`swap()`](https://doc.rust-lang.org/std/sync/atomic/struct.AtomicU64.html#method.swap) may be better than `load` + `store`